### DISCUSSION TITLE
Welcome to llama-rs Discussions!

### DISCUSSION THREAD

**Discussion by philpax** [rustformers/llama-rs#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juho-p** [comment]
Cool project, and awesome work so far :D I've been playing around it occasionally for couple of weeks now (the library part that is, though I haven't built anything useful with it yet). Much much easier to use API than the llama.cpp C-API that they provide nowdays.
